mobile-app/lib/ui/views/learn/utils/learn_globals.dart

```diff
@@ -7,3 +7,10 @@ List<String> hasDialogue = [
   'a2-english-for-developers',
   'b1-english-for-developers',
 ];
+
+const chapterBasedSuperBlocks = [
+  'full-stack-developer',
```

Comment:
I think we can exclude this one as we're going to remove it: https://github.com/freeCodeCamp/freeCodeCamp/pull/64283.

```suggestion
```